File: java/claude/P14.java
// Task p14: Function takes index, checks bounds, returns value or -1, returns 0 if successful
import java.util.Scanner;

public class P14 {
    /**
     * Takes index, checks if in bounds, returns value or -1.
     */
    public static int getValueFromArray(int index) {
        int[] idSequence = {10, 20, 30, 40, 50, 60, 70, 80, 90, 100};
        int size = idSequence.length;

        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter an index: ");

        try {
            int userIndex = scanner.nextInt();

            if (userIndex < 0 || userIndex >= size) {
                System.out.println("Result: -1");
            } else {
                System.out.println("Result: " + idSequence[userIndex]);
            }
        } catch (Exception e) {
            System.out.println("Invalid input.");
        }

        scanner.close();
        return 0;
    }

    public static void main(String[] args) {
        getValueFromArray(0);
    }
}

File: python/claude/p14.py
# Task p14: Function takes index, checks bounds, returns value or -1, returns 0 if successful
def get_value_from_array(index):
    """
    Takes index, checks if in bounds, returns value or -1.
    """
    id_sequence = [10, 20, 30, 40, 50, 60, 70, 80, 90, 100]
    size = len(id_sequence)

    try:
        user_index = int(input("Enter an index: "))

        if user_index < 0 or user_index >= size:
            return -1
        return id_sequence[user_index]
    except ValueError:
        return -1


if __name__ == "__main__":
    result = get_value_from_array(0)
    print(f"Result: {result}")
